docs/source/command_line.rst

```diff
@@ -342,6 +342,9 @@ Here are some more useful flags:
   when most parts of your program haven't changed since the previous
   mypy run.
 
+  Incremental mode is now the default and may be disabled with
```

Comment:
This is not a news entry, I'd leave "now" off.

Also could you regenerate the option summary at the top of this file? There's a `[-i]` tucked in there somewhere that probably should go, and `[--no-incremental]` should appear.